### PR TITLE
Feature/api table styles

### PR DESCRIPTION
**Pull request by emclaug2**
#### Changes proposed in this Pull Request:
- Make the tables look pretty

### CODE REVIEW

File: src/themes/doc-light.scss [hunk 1]
```diff
@@ -9,4 +9,33 @@
...
+        table {
+            * {
+                box-sizing: border-box;
+            }
+            tr th {
+                text-align: left;
+                margin: 0px;
+                padding: 6px 13px;
+                font-weight: bold;
+                border: 1px solid rgb(213, 216, 218);
```
Comment (emclaug2) [inline src/themes/doc-light.scss:33]: Eh, this should probably be a from the colors palette.

**Comment by huayunh** [comment]
There are quite a few places where the markdown did not get interpreted correctly. I don't know if this is within the scope of this PR though

image

**Comment by emclaug2** [comment]
> There are quite a few places where the markdown did not get interpreted correctly. I don't know if this is within the scope of this PR though
> 
> image

I'd say fixing anything table-related can be a part of this story, yes.  But 'primary' 'accent' and 'warn' are strings which is why they appear quoted instead of as a type `boolean` or `undefined`.

I see 'primary' is displayed as a string & a type in 2 separate places, I'll update it to be a string.

**Comment by emclaug2** [comment]
> > There are quite a few places where the markdown did not get interpreted correctly. I don't know if this is within the scope of this PR though
> 
> I'd say fixing anything table-related can be a part of this story, yes. But 'primary' 'accent' and 'warn' are strings which is why they appear quoted instead of as a type `boolean` or `undefined`.
> 
> I see 'primary' is displayed as a string & a type in 2 separate places, I'll update it to be a string.

**Comment by emclaug2** [comment]
@huayunh I just noticed what you pointed out, the table is all wonky - I'll patch that.

**Comment by huayunh** [comment]
If the goal is to make the table look exactly what we specify in our design docs,

image

**Comment by huayunh** [comment]
Other than that, i think your table looks good!

I'd say this thing does not belong in the "API  Docs" tab, as it should be shown under "Example". But that is out of the scope
image

**Comment by emclaug2** [comment]
Thanks @huayunh , PR has been updated.

**Review by huayunh — APPROVED** [review]
lgtm 

(I didn't check the code)

**Pull request merged**